src/Aspire.Cli/DotNet/IDotNetRuntimeSelector.cs

```diff
@@ -0,0 +1,54 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Aspire.Cli.DotNet;
+
+/// <summary>
+/// Service responsible for selecting and managing the .NET runtime to use.
+/// </summary>
+internal interface IDotNetRuntimeSelector
+{
+    /// <summary>
+    /// Gets the path to the dotnet executable to use.
+    /// </summary>
+    string GetDotNetExecutablePath();
```

Comment:
Make this `Task<string> GetDotNetExecutablePathAsync` so we can install the SDK cleanly.